secureboot_policy.go

```diff
@@ -711,7 +730,12 @@ type authenticodeSignerAndIntermediates struct {
 // secureBootPolicyGen is the main structure involved with computing secure boot policy PCR digests. It is essentially just
 // a container for EFISecureBootPolicyProfileParams - per-branch context is maintained in secureBootPolicyGenBranch instead.
 type secureBootPolicyGen struct {
-	*EFISecureBootPolicyProfileParams
+	pcrAlgorithm  tpm2.HashAlgorithmId
+	loadSequences []*EFIImageLoadEvent
+
+	events                     []*tcglog.Event
+	initialOSVerificationEvent *secureBootVerificationEvent
+	sigDbUpdates               []*secureBootDbUpdate
```

Comment:
no strong opinion, but could this also carry the quirk mode?

then the branches can get it via  branch.gen.... ?